spec/requests/message_handler_spec.rb

```diff
@@ -1,7 +1,30 @@
 require 'rails_helper'
 
 describe "Logging API requests" do
-  context "Page Creation" do
+  let(:page) { Page.create(title: 'Foo', slug: 'foo-bar') }
+
+  context "Updating Page Resource" do
+    let(:params) do
+      { type: 'update',
+        uri: "https://act.sumofus.org/rest/v1/petitionpage/8769/",
+        params: {
+          title: 'Foo Bar',
+          tags: ["/rest/v1/tag/878/"]
+        }
+      }
+    end
+
+    context "204 response" do
+      it "updates resource" do
+        VCR.use_cassette('update_resource_204') do
```

Comment:
`VCR.user_cassette` is the API I know. Sounds like it does the same thing at `:vcr`